Nim/Program.cs

```diff
@@ -0,0 +1,42 @@
+﻿Console.WriteLine("Welcome to Nim!\nLet's play: we take turns drawing matches, and the one who draws the last one is the loser.");
+int matches = 24;
+string draw = "||||||||||||||||||||||||";
+Console.WriteLine($"{draw} ({matches})");
```

Comment:
You can use the same method as below: `Console.WriteLine($"The number of matches is: {new string('|', matches)} ({matches})");`